plugins/producer_api_plugin/producer_api_plugin.cpp

```diff
@@ -114,16 +112,12 @@ void producer_api_plugin::plugin_startup() {
             INVOKE_R_V(producer, get_greylist), 201),
        CALL_WITH_400(producer, producer, get_whitelist_blacklist,
             INVOKE_R_V(producer, get_whitelist_blacklist), 201),
-       CALL_WITH_400(producer, producer, set_whitelist_blacklist,
-            INVOKE_V_R(producer, set_whitelist_blacklist, producer_plugin::whitelist_blacklist), 201),
        CALL_WITH_400(producer, producer, get_integrity_hash,
             INVOKE_R_V(producer, get_integrity_hash), 201),
        CALL_ASYNC(producer, producer, create_snapshot, producer_plugin::snapshot_information,
             INVOKE_R_V_ASYNC(producer, create_snapshot), 201),
```

Comment:
`create_snapshot` not safe when running read-only trxs when in IRREVERSIBLE mode as it `abort_block()` and `schedule_production_loop()`.